### ISSUE TITLE
[DOCTRINE] Not update Schema after use json type field

### ISSUE THREAD

**Issue by orisha** [symfony/symfony#31740]
**Symfony version(s) affected**: 3.4, doctrine: 

**Description**  
I can not update a Schema based on changes done on Entity files !
I created for test purposes a new entity within a 'json' typed field.

First, it worked fine: I was able to schema update, store and read on/for the field.

But, when I modify the Entity file and try to update the schema by terminal, it throws an error.
Like, if I remove a field, it throws an error say that the field doens't exists . ( Sure it doens't exists, I've modified it )
Same happens if I delete the entity file: the error says that the class doesn't exists

This error only happens in my localhost ( XAAMP on MacOS 10.14 ).
I also have an vps runnning ubuntu whrere the problem does NOT show !!!

Sure I tried many times to clear caches from Symfony and Doctrine. Even tried to delete cookies, restart the apache/mysql and the computer ! Always same error ...
Also tried to drop and rename database ... 

**How to reproduce**  
I can only get this error on my localhost via terminal:
php bin/console doctrine:schema:update --force --em=MyTest

....

This error start to show only AFTER include 'json' type !!!! Never used such type before, never got similar problem before

**Comment by xabbuh** [comment]
Doctrine ORM is not part of the Symfony core. It is managed in its [own repository](https://github.com/doctrine/doctrine2/). If you think there is a bug in it, please report it over there providing some details on how the maintainers would be able to reproduce it. I am closing here as there isn't anything we can do about it here. Thank you for understanding.
